addbookfunction.php

```diff
@@ -2,12 +2,14 @@
 require 'functions.php';
 $db = startDb('books');
 
-if (array_key_exists('bookName', $_POST)
-    && array_key_exists('author', $_POST)
-    && array_key_exists('category', $_POST)
-    && array_key_exists('released', $_POST)) {
-    $query = $db->prepare('INSERT INTO `booksCollected` (`name`, `author`, `category`, `released`, `deleted`) VALUES (?, ?, ?, ?, ?)');
-    if ($query->execute([$_POST['bookName'], $_POST['author'], $_POST['category'], $_POST['released'], 0])) {
+if (
+    array_key_exists('bookName', $_POST) &&
+    array_key_exists('author', $_POST) &&
+    array_key_exists('category', $_POST) &&
+    array_key_exists('released', $_POST)
+) {
+    $query = $db->prepare('INSERT INTO `booksCollected` (`name`, `author`, `category`, `released`, `deleted`) VALUES (:bookname, :author, :category, :releaseYear , :deleteStatus)');
+    if ($query->execute([':bookname' => $_POST['bookName'], ':author' => $_POST['author'], ':category' => $_POST['category'], ':releaseYear' => $_POST['released'], ':deleteStatus' => 0])) {
```

Comment:
you are taking values from an array and putting them into an almost identical array. why not use the `$_POST` array and set the named placeholders to the same keys